lib/services/eventgridManagement/lib/models/topic.js

```diff
@@ -43,30 +43,6 @@ class Topic extends models['TrackedResource'] {
         name: 'Composite',
         className: 'Topic',
         modelProperties: {
-          id: {
-            required: false,
-            readOnly: true,
-            serializedName: 'id',
-            type: {
-              name: 'String'
-            }
-          },
-          name: {
-            required: false,
-            readOnly: true,
-            serializedName: 'name',
-            type: {
-              name: 'String'
-            }
-          },
-          type: {
-            required: false,
-            readOnly: true,
-            serializedName: 'type',
-            type: {
-              name: 'String'
-            }
-          },
```

Comment:
It looks like when the base type changed from `Resource` to `BaseResource` we lost these properties. If that's the case, then this is a breaking change and needs a major version bump.